deb/debian/rules

```diff
@@ -20,7 +20,7 @@ ifeq ($(DISTRO),xenial)
 DOCKER_FLAG="--enable-docker"
 endif
 
-ifeq ($(DISTRO),bionic)
+ifneq (,$(filter $(DISTRO),bionic buster))
 # Use system libmongoc on Ubuntu Bionic.
```

Comment:
```suggestion
 # Use system libmongoc.
```

Reply:
Suggest modifying the comment in this way, otherwise we need to update the comments every time we add new distro. What do you think?

Reply:
Done, with a bit more explanation.